test-tools/dynamic-aot-debug/README.md

```diff
@@ -0,0 +1,59 @@
+# Dynamic AOT Module Debugging
+
+### How to use
+
+#### 1. Enable config
+
+```
+WASM_ENABLE_AOT=1
+WASM_ENABLE_DYNAMIC_AOT_DEBUG=1
+```
+
+#### 2. Build aot and obj file
+
+- build [wamrc](https://github.com/bytecodealliance/wasm-micro-runtime/tree/main/wamr-compiler) (enable WAMR_BUILD_DEBUG_AOT)
+- build test.c to test.wasm
+
+```
+/opt/wasi-sdk/bin/clang -O0 -nostdlib -z stack-size=8192 -Wl,--initial-memory=65536 -g -gdwarf-2   -o test.wasm test.c -Wl,--export=main -Wl,--export=__main_argc_argv  -Wl,--export=__heap_base -Wl,--export=__data_end -Wl,--no-entry -Wl,--allow-undefined
+```
+
+- build test.wasm to test.aot
+
+Compile and generate the test.aot file using wamrc without WAMR_BUILD_DEBUG_AOT.
+
+```
+./wamrc --opt-level=0 --target=thumbv7 --target-abi=gnueabihf--cpu=cortex-a7 --cpu-features=-neon -o test.aot test.wasm
+```
+
+- build test.wasm to test obj file
+
+Use wamrc that needs to turn on WAMR_BUILD_DEBUG_AOT to compile and generate the test.aot file.
+
+```
+./wamrc --opt-level=0 --format=object --target=thumbv7 --target-abi=gnueabihf --cpu=cortex-a7 --cpu-features=-neon -o test test.wasm
+```
+
+#### 3. Start emulator and nuttx
```

Comment:
It should support natvie iwasm or other platform, add lease the native iwasm guide.

Reply:
OK

Reply:
Yes, it would be highly appreciated if a document and an example are provided for native iwasm. For example, in Ubuntu/MacOS, compile a simple C app to wasm and then to the AOT file, and then demonstrate how to set the break point at a line of C source and view or dump the call stack.